tests/components/homewizard/snapshots/test_sensor.ambr

```diff
@@ -6468,15 +6468,15 @@
     'supported_features': 0,
     'translation_key': 'active_liter_lpm',
     'unique_id': 'HWE-P1_5c2fafabcdef_active_liter_lpm',
-    'unit_of_measurement': 'l/min',
+    'unit_of_measurement': <UnitOfVolumeFlowRate.LITERS_PER_MINUTE: 'L/min'>,
```

Comment:
Is this a breaking change?
Will it break long-term statistics?

I ask because of the change in casing ( `l` vs `L` ) and I'm not sure if that is handled automatically or not so maybe it needs to be marked as breaking.

Reply:
It is not intended as a breaking change, if it does I would prefer to not merge this PR or or add a migration.

But I've been told this will trigger a repair issue, which should fix LTS. I don't known if this is correct.